mantidproject/mantid
# Unicode workspace name

Allow for people to use unicode when requesting workspaces in scripts with python2. 

**To test:**

There are two scripts to try out (python2 will show an error, python3 auto-converts already):
```python
from __future__ import (absolute_import, division, print_function, unicode_literals)
from mantid.simpleapi import *

CreateSampleWorkspace(OutputWorkspace='w')
w = mtd['w']
print(w)
```
and
```python
from mantid.simpleapi import *

CreateSampleWorkspace(OutputWorkspace='w')
w = mtd[unicode('w')]
```
Both of which should work.

*There is no associated issue.*

*Does not need to be in the release notes.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
